config/prow/config_knative.yaml

```diff
@@ -915,6 +924,18 @@ periodics:
       go113: true
       env-vars:
       - ORG_NAME=google
+    - continuous: true
```

Comment:
I think you are missing a heading here.

Reply:
Good catch. I should trust the suggested commit...